checker/src/org/checkerframework/checker/interning/qual/Interned.java

```diff
@@ -26,16 +28,18 @@
  * annotated.
  * <p>
  *
- * This annotation is associated with the {@link InterningChecker}.
+ * This annotation is associated with the {@link org.checkerframework.checker.interning.InterningChecker}.
```

Comment:
My IDE made this change when auto-organizing the imports.  I think I have now changed the setting so that it will use simple names in JavaDoc and add the correct import.  (I'll also change this back when I address your other feedback.)